### PR TITLE
Release v0.13.0

### PR DESCRIPTION
**Pull request by SuperQ**
BREAKING CHANGES:

Please note, the following features are deprecated and may be removed in a future release:
- `auto-discover-databases`
- `extend.query-path`
- `constantLabels`
- `exclude-databases`
- `include-databases`

This exporter is meant to monitor PostgresSQL servers, not the user data/databases. If you need a generic SQL report exporter https://github.com/burningalchemist/sql_exporter is recommended.

* [CHANGE] Adjust log level for collector startup #784
* [CHANGE] Move queries from queries.yaml to collectors #801
* [CHANGE] Deprecate extend queries feature #811
* [CHANGE] Deprecate additional database features #815
* [CHANGE] Convert pg_stat_database to new collector #685
* [ENHANCEMENT] Supports alternate postgres:// prefix in URLs #787
* [BUGFIX] Fix pg_setting different help values #771
* [BUGFIX] Fix column type for pg_replication_slots #777
* [BUGFIX] Fix pg_stat_database collector #809

### CODE REVIEW

**Comment by SuperQ** [comment]
:eyeglasses: @Sticksman

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,27 @@
-## master / unreleased
+## 0.13.0 / 2023-06-21
```
Comment (sysadmind) [inline CHANGELOG.md:1]: Not feedback on this specific PR, but something to think about in general: I think we could stop posting the dates here. The github release page will have the release date and not having dates gives us the flexibility for reviews to take a few days if need be.

**Review by sysadmind — APPROVED** [review]
LGTM do we have any dependencies to update for the release?

File: CHANGELOG.md (hunk 1 above)
Comment (SuperQ) [reply CHANGELOG.md:1]: The date thing is a promu requirement. I guess we would have to discuss that with the rest of prometheus-dev.

**Comment by SuperQ** [comment]
I think the last round of dependabot PRs got all of the Go mods updated.

**Review by Sticksman — COMMENTED** [review]
Lgtm!

**Pull request merged**